docs/source/reference/builtin-devices.rst

```diff
@@ -40,22 +40,60 @@ This must be sub-classed (like :class:`~ophyd.device.Device`) to be useful.
    ophyd.epics_motor.EpicsMotor
    ophyd.epics_motor.MotorBundle
 
+.. _built_in.EpicsScaler:
+
 EpicsScaler
 -----------
 
 Create an ``EpicsScaler`` to control an EPICS `scaler record
-<http://www.aps.anl.gov/bcda/synApps/std/scalerRecord.html>`_:
+<https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_:
 
 .. code-block:: python
 
     from ophyd import EpicsScaler
     scaler = EpicsScaler('XF:28IDC-ES:1{Sclr:1}', name='tth')
 
+Note that :ref:`built_in.ScalerCH` is an alternative ``ophyd`` representation of
+the EPICS `scaler record
+<https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_.
+
 .. autosummary::
    :toctree: ../generated
 
    ophyd.scaler.EpicsScaler
 
+.. _built_in.ScalerCH:
+
+ScalerCH
+--------
+
+Create a ``ScalerCH`` to control an EPICS `scaler record
+<https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_:
+
+.. code-block:: python
+
+    from ophyd import ScalerCH
+    scaler = ScalerCH('XF:28IDC-ES:1{Sclr:1}', name='tth')
+
+Note that :ref:`EpicsScaler` is an alternative ``ophyd`` representation of
+the EPICS `scaler record.
```

Comment:
```suggestion
the EPICS `scaler record <https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_.
```

Reply:
The URL grew too complicated to expect that further edits will not become problems.  I'm refactoring the scaler support into a *EPICS scaler record* section with subsections for *EpicsScaler* and *ScalerCH*.  People need guidance about which to choose (since there are two) and that should come first.